test/ring/middleware/session_timeout_test.clj

```diff
@@ -17,21 +17,42 @@
   {:timeout 600
    :timeout-response timeout-response})
 
+(defn- make-handler
+  [response]
+  (fn
+    ([_] response)
+    ([_ cont _] (cont response))))
```

Comment:
Use `respond` instead of `cont`.

Reply:
Also, don't use a handler that has both synchronous and asynchronous arities. If you're testing asynchronous handlers, just use the asynchronous arity. This will expose more potential bugs.